app/main.py

```
@@ -1 +1,28 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None | str:
+    if len(command.split()) != 3:
+        return "Incorrect statement printed"
+
+    cmd, curent_file, destination = command.split()
+
+    if cmd != "mv":
+        return "Command is incorrect"
+
+    dir_list = destination.split("/")
+    new_file_name = dir_list[-1]
+    dir_list.remove(new_file_name)
+
+    try:
+        os.makedirs("/".join(dir_list))
+    except FileExistsError:
+        print("Folder is already exists")
```

Use os.makedirs(path, exist_ok=True) to ignore this exceptions